### PR TITLE
Remove all Kotlin code and ISI resources

### LINKED ISSUE #108: Deprecate ColdStart in AIDA
In the upcoming [Java version of the AIF Mapper](https://github.com/NextCenturyCorporation/AIDA-Interchange-Format/issues/107), ColdStart will not be supported, but the Kotlin source will remain in the source tree.

See Next Century's Jira Issue [AIDA-468](https://nextcentury.atlassian.net/browse/AIDA-468).

### PR DESCRIPTION
**Pull request by berquist**
Closes #108.

I didn't scour the `pom.xml` for extra dependencies, but `git clean -xdf .; rm -r ~/.m2; mvn install` works.

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -26,25 +26,3 @@ relation 2 or relation 3") since we assume that such cases should expressed as t
 distinct hypotheses (e.g. one hypothesis for "relation 1 and relation 2" and another for
 "relation 1 and relation 3").  However, if expressing such disjunctions is desired, it can be
 added.
-
```
Comment (rgabbard) [inline FAQ.md]: This should probably stay?

File: pom.xml [hunk 2]
```diff
@@ -44,17 +38,6 @@
...
-            <groupId>org.jetbrains.kotlin</groupId>
-            <artifactId>kotlin-stdlib-jdk8</artifactId>
-            <version>${kotlin.version}</version>
-        </dependency>
-        <dependency>
-            <groupId>org.jetbrains.kotlin</groupId>
-            <artifactId>kotlin-test</artifactId>
-            <version>${kotlin.version}</version>
-            <scope>test</scope>
-        </dependency>
```
Comment (nextcen-dgemoets) [inline pom.xml]: I believe you can remove edu.isi.vista's common-core-open dependency.

File: README.md [hunk 3]
```diff
@@ -6,36 +6,27 @@ This repository contains resources to support the AIDA Interchange Format (AIF).
...
      `src/main/resources/com/ncc/aif/aida_ontology.shacl`.
 
-*    utilities to make it easier to work with this format.  Java utilities are in
-     `src/main/java/com/ncc/aif/AIFUtils.java`. A Kotlin version of the utilities are in
-	 `src/main/java/edu/isi/gaia/AIFUtils.kt`.  Either of these can be used by adding a Maven dependency on
-     `com.ncc:aida-interchange:1.0.0-SNAPSHOT`.  A Python translation of these utilities
-     is in `python/aida_interchange/aifutils.py`.
+*    utilities to make it easier to work with this format.  Java utilities are
+     in `src/main/java/com/ncc/aif/AIFUtils.java`, which can be used by adding
+     a Maven dependency on `com.ncc:aida-interchange:1.0.0-SNAPSHOT`.  A
```
Comment (nextcen-dgemoets) [inline README.md]: Could you change this to 1.1.0-SNAPSHOT?  We missed that when we updated the version.

**Comment by rgabbard** [comment]
@craigwarsaw : This looks good from our end.

File: README.md [hunk 4]
```diff
@@ -47,7 +38,7 @@ example, include the following in the dependencies in your `build.gradle` file:
     compile 'com.ncc:aida-interchange:1.0-SNAPSHOT'
```
Comment (nextcen-dgemoets) [inline README.md]: Also 1.1.0-SNAPSHOT here.

**Comment by berquist** [comment]
Done.

**Comment by nextcen-dgemoets** [comment]
Sorry I missed it earlier, but could you remove the reference to ISI Vista Common Core Open in the `README.txt`?  Should be my last comment.

**Comment by berquist** [comment]
There is also `Copyright (c) 2018 isi-vista` in the top-level `LICENSE`. What should be done about that?

**Comment by nextcen-dgemoets** [comment]
> There is also Copyright (c) 2018 isi-vista in the top-level LICENSE. What should be done about that?

I don't know... @craigwarsaw do you have any thoughts on this?

**Comment by rgabbard** [comment]
@berquist : I would recommend just deleting it for this PR.  NCC can add their own updated LICENSE info at their convenience.

**Comment by berquist** [comment]
Ok, I've removed it and someone can poke me if they'd like that change reverted.

**Comment by craigwarsaw** [comment]
> > There is also Copyright (c) 2018 isi-vista in the top-level LICENSE. What should be done about that?
> 
> I don't know... @craigwarsaw do you have any thoughts on this?

Thanks @ericholgate and @nextcen-dgemoets for collaborating on this PR.
We do need a LICENSE here so we shouldn't merge without a LICENSE.  Removing the Kotlin code shouldn't force users of this library to accept new license terms, so we should continue to use the MIT license - https://opensource.org/licenses/MIT.  That license requires a copyright holder.  We shouldn't assert Next Century copyright if ISI still wishes to assert copyright.  Please wait to hear from me before we merge this to master.  Thanks again!

**Comment by nextcen-dgemoets** [comment]
We are still working on the license issue on our end.  So if you want to proceed with getting this PR approved and merged, you can put the `LICENSE` file back into the tree as it was originally, and we can address/resolve the license issue separately down the road.  Or if you prefer we can wait on this PR until the license issue is resolved.

**Comment by berquist** [comment]
I've put the license back so that issue can be resolved later.

**Comment by rgabbard** [comment]
@craigwarsaw @nextcen-dgemoets : Thanks for merging this.  Would it be possible to make a release so we can depend on it in Maven in a less error-prone way than via snapshots?

**Comment by nextcen-dgemoets** [comment]
@rgabbard  We've created a ticket for this and added it to our upcoming Sprint (which starts this afternoon).

**Comment by nextcen-dgemoets** [comment]
@rgabbard  This change has made its way into our upcoming v1.0.1 release.  We will announce its release late this week / early next week.

**Comment by nextcen-dgemoets** [comment]
@rgabbard  This has been incorporated into [today's release](https://github.com/NextCenturyCorporation/AIDA-Interchange-Format/releases).

**Pull request merged**